issue/issue.go

```diff
@@ -97,17 +97,18 @@ var ruleToCWE = map[string]string{
 
 // Issue is returned by a gosec rule if it discovers an issue with the scanned code.
 type Issue struct {
-	Severity     Score             `json:"severity"`     // issue severity (how problematic it is)
-	Confidence   Score             `json:"confidence"`   // issue confidence (how sure we are we found it)
-	Cwe          *cwe.Weakness     `json:"cwe"`          // Cwe associated with RuleID
-	RuleID       string            `json:"rule_id"`      // Human readable explanation
-	What         string            `json:"details"`      // Human readable explanation
-	File         string            `json:"file"`         // File name we found it in
-	Code         string            `json:"code"`         // Impacted code line
-	Line         string            `json:"line"`         // Line number in file
-	Col          string            `json:"column"`       // Column number in line
-	NoSec        bool              `json:"nosec"`        // true if the issue is nosec
-	Suppressions []SuppressionInfo `json:"suppressions"` // Suppression info of the issue
+	Severity     Score             `json:"severity"`          // issue severity (how problematic it is)
+	Confidence   Score             `json:"confidence"`        // issue confidence (how sure we are we found it)
+	Cwe          *cwe.Weakness     `json:"cwe"`               // Cwe associated with RuleID
```

Comment:
```suggestion
	CWE          *cwe.Weakness     `json:"cwe"`               // CWE associated with RuleID
```

CWE is an acronym for Common Weakness Enumeration

Acronym are in uppercase

Reply:
@ccojocar What do you think